package.json

```diff
@@ -40,6 +40,6 @@
   },
   "dependencies": {
     "commander": "^4.1.1",
-    "puppeteer-lottie": "github:adpipe/puppeteer-lottie"
+    "puppeteer-lottie": "github:adpipe/puppeteer-lottie#06487ee3c4e9aaf399443f281a7058cd7c18d192"
```

Comment:
should we be using `#master` instead of specifying a commit?

Reply:
honestly I was thinking about going in the opposite direction and specifying sha at both the CLI and smallville level, to facilitate easy rollback. thoughts?

Reply:
oooh how does it facilitate easy rollback? I thought you'd have to come in here and change the commit SHAs anyway

Reply:
I guess I mean moving forward, the next change can just be a sha change in smallville, which will pickup the correct sha in `puppeteer-lottie-cli` which will pick up the correct sha in `puppeteer-lottie`. then rolling back the sha specification will ensure that we're not randomly caching a version pin. If i merge this change now, the next smallville version will pick it up, because right now it's picking master.

Reply:
ohhh yeah i see what you're saying that makes sense